src/index.html

```diff
@@ -8,6 +8,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+
+      <li class="stars__star">
+      </li>
+    </ul>
```

Comment:
let's keep the code style cleaner
```suggestion
    <ul class="stars stars--0">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
```